src/ia-dropdown.ts

```diff
@@ -70,6 +70,17 @@ export class IaDropdown extends LitElement {
 
   @property({ type: Function }) optionSelected = () => {};
 
+  @property({ type: Boolean, reflect: true }) isCustomList = false;
+
+  @query('.click-main') mainButton!: HTMLButtonElement;
+
+  async firstUpdated(): Promise<void> {
+    await new Promise(resolve => setTimeout(resolve, 0));
+    this.addEventListener('closeDropdown', () => {
+      this.open = false;
+    });
+  }
```

Comment:
This code is in the LIT docs.
https://lit.dev/docs/components/events/#async-events
This might not be the best example where there's a need to wait until render done before listening for event.

Reply:
> `connectedCallback`

OMG thanks! Not here, but on NON-element events it should be used. I will fix those.
https://lit.dev/docs/components/events/#adding-event-listeners-to-other-elements